tensorflow/compiler/xla/service/llvm_ir/ir_builder_mixin.h

```diff
@@ -110,6 +110,11 @@ class IrBuilderMixin {
     return mixin_builder()->CreateFAdd(std::forward<Args>(args)...);
   }
 
+  template <class... Args>
+  llvm::Value* Fence(Args&&... args) {
```

Comment:
Mixins generally make code a lot harder to read, maybe just call b->CreateFence?

Reply:
Done.